signer/src/bitcoin/utxo.rs

```diff
@@ -1425,13 +1430,24 @@ pub trait TxDeconstructor: BitcoinInputsOutputs {
             .collect()
     }
 
+    /// Return all outputs in this transaction that are related to the signers
+    /// and any relevant withdrawal output.
+    fn to_outputs(
```

Comment:
Thought: I might have skipped this tbh, since where you use it you seem to iterate over them separately anyway.

Reply:
That's true, but `to_withdrawal_outputs` depends on `to_tx_outputs` and I didn't want to call it twice if we wanted both anyways.

Reply:
Ah, right 👍